modules/tinymce/src/plugins/link/main/ts/core/DetectRtc.ts

```diff
@@ -9,7 +9,7 @@ import Editor from 'tinymce/core/api/Editor';
 import PluginManager from 'tinymce/core/api/PluginManager';
 
 export const hasRtcPlugin = (editor: Editor) => {
-  if (/(^|[ ,])rtc([, ]|$)/.test(editor.settings.plugins) && PluginManager.get('rtc')) {
+  if (/(^|[ ,])rtc([, ]|$)/.test(editor.getParam('plugins', '', 'string')) && PluginManager.get('rtc')) {
```

Comment:
Wasn't there a comment or something above about how `plugins` can take an array? Hm. Could you please check that and adjust this if necessary?

Reply:
`Render.ts` apparently

Reply:
Considering I'm finding multiple instances of this, if `plugins` can be an array it might make more sense to Jira up fixing all the references. I'm sure there's many of them. I may have misunderstood the comment I'm referring to though...

Reply:
IIRC EditorSettings.ts converts it to a single string somewhere in the init sequence.

Reply:
See https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/api/Editor.ts#L293 and https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/EditorSettings.ts#L180 so I suspect the other place where it's checking to see if it's an array is incorrect.